src/lib/components/design_tile/DesignTile.js

```diff
@@ -20,18 +20,14 @@ export default class DesignTile extends Component {
 
     const classes = classnames(CLASS_ROOT, className, 'teepublic');
 
-    const { images, price } = designSku;
-
-    var skuMockupImage = _.find(images, function(image) {
-      return image.type === 'mockup';
-    });
+    const { defaultImage, price } = designSku;
```

Comment:
Does defaultImage look for a mockup and then fallback to a design image?

Reply:
It looks for a mockup, front, left, then falls back the the first image that is returned by `images_for_product` in the TeePublic app.

Reply:
Okay cook, I approve, way better